test/cli/index_test.cpp

```diff
@@ -0,0 +1,133 @@
+#include <fstream>
+#include <string>
+
+#include "cli_test.hpp"
+
+struct index_test : public cli_test
+{
+    void run_index_test(std::string const & index_command,
+                        std::string const & db_file,
+                        std::string const & index_file,
+                        std::string const & index_type,
+                        std::string const & reduction,
+                        std::string const & control_file)
```

Comment:
It's good style to use string_view in modern C++, but if changing it breaks something, don't bother :)

```suggestion
                        std::string_view const control_file)
```
(also in other places)

Reply:
This causes many things to break, so I will not bother as suggested :slightly_smiling_face: